index.html

```diff
@@ -416,15 +423,21 @@ <h4>Audio Description Examples</h4>
         <h4>Dubbing Examples</h4>
         <p>From the basic structure of <a href="#example-1"></a>, a transcription of the audio programme produces an original language dubbing <a>script</a>,
```

Comment:
minor editorial: the hyperlink is on the word script. It does not feel right because a transcription would be a transcript. I suggest either putting the hyperlink around "dubbing script" or changing script to transcript.

Reply:
Will change to transcript, and fix "dialogs" below, for consistency, because we use "dialogue" everywhere else.

Reply:
Done in a1c80d4